etc/kayobe/kolla/config/prometheus/prometheus-alertmanager.yml

```diff
@@ -0,0 +1,32 @@
+{% raw %}
+global:
+  resolve_timeout: 5m
+  smtp_require_tls: true
+  slack_api_url: 'https://hooks.slack.com/services/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'
```

Comment:
Does this URL contain a secret? Can anyone post messages to this slack channel now?

Reply:
Good point, forgot this was public, I will fix that.